ci-operator/step-registry/ipi/conf/azure/ipi-conf-azure-commands.sh

```diff
@@ -60,3 +59,28 @@ EOF
     echo "${OUTBOUND_TYPE} is not supported yet" || exit 1
   fi
 fi
+
+set +u
```

Comment:
suggest to remove the line.

Reply:
in order to avoid error when has not defined var, I add this. 
PUBLISH and OCP_VERSION maybe not defined.